Docs: Add Installation YouTube Videos to documentation pages. (#88465)
grafana/grafana: docs/sources/setup-grafana/installation/mac/index.md

This page explains how to install Grafana on macOS.

> The following video guides you through the steps, common commands, and how to install Grafana in macOS as described in this document:

Reviewer: ```suggestion
The following video demonstrates how to install Grafana on macOS as outlined in this document:
```
